pt/lessons/specifics/nerves.md

```diff
@@ -0,0 +1,262 @@
+---
+version: 1.0.1
+title: Nerves
+redirect_from:
+  - /pt/lessons/advanced/nerves
+---
+
+{% include toc.html %}
+
+## Introdução e requisitos
+
+Nós falaremos sobre Nerves nessa lição. O projeto Nerves é um framework para utilizar Elixir em desenvolvimento de software embarcado. Como o website do Nerves diz, ele permite que você "construa e faça deploy de software embarcado à prova de balas em Elixir". Essa lição será um pouco diferente de outras lições da Elixir School. Nerves é um pouco mais difícil de absorver já que requer a configuração de um sistema avançado e hardware adicional, então pode não ser adequado para iniciantes.
+
+Para escrever código embarcado usando Nerves, você vai precisar de um dos [dispositivos suportados](https://hexdocs.pm/nerves/targets.html), um leitor de cartão com um cartão de memória suportado pelo hardware de sua escolha, e uma conexão a rede cabeada para acessar esse dispositivo pela rede.
+
+De qualquer forma, nós sugerimos usar um Raspberry Pi, já que ele tem um LED controlável onboard. Também é recomendável ter uma tela conectada ao seu dispositivo alvo já que isso vai simplificar o debug usando IEx.
+However, we would suggest using a Raspberry Pi, due to it having controllable LED onboard. It is also advisable to have a screen connected to your target device as this will simplify debugging using IEx.
+
+## Configuração
+
+O projeto Nerves tem um excelente [Guia de introdução](https://hexdocs.pm/nerves/getting-started.html), mas a quantidade de detalhes lá pode ser assustadora para alguns usuários. Em vez disso, esse tutorial vai tentar apresentar "menos palavras, mais código".
```

Comment:
I little confusing between these two words `tutorial` or `lição`, what kind it could be more fit to use?

Reply:
I actually used those because both are being used throughout Elixir School, sometimes in the same. What I could grasp is that `tutorial` is being used by some lessons with this structure; while `lição` is being used to refer to Elixir School's pages in general.
I do think both shall be used since they refer to different things.